### ISSUE TITLE
Missing NodeSelector for "functioning NodePort" tests for windows

### ISSUE THREAD

**Issue by jayunit100** [kubernetes/kubernetes#97807]
**What happened**:

Similar to #97805 , were missing the windows nodeselectors for this one...  resulting in failed pod sandboxes due to missing metadata, root cause is easy fix, will issue PR shortly

```
spec:
  volumes:
    - name: default-token-9c9ts
      secret:
        secretName: default-token-9c9ts
        defaultMode: 420
  containers:
    - name: netexec
      image: 'k8sprow.azurecr.io/kubernetes-e2e-test-images/agnhost:2.21'
      args:
        - netexec
        - '--http-port=80'
        - '--udp-port=80'
      resources: {}
      volumeMounts:
        - name: default-token-9c9ts
          readOnly: true
          mountPath: /var/run/secrets/kubernetes.io/serviceaccount
      readinessProbe:
        httpGet:
          path: /hostName
          port: 80
          scheme: HTTP
        timeoutSeconds: 1
        periodSeconds: 3
        successThreshold: 1
        failureThreshold: 3
      terminationMessagePath: /dev/termination-log
      terminationMessagePolicy: File
      imagePullPolicy: IfNotPresent
  restartPolicy: Always
  terminationGracePeriodSeconds: 3
  dnsPolicy: ClusterFirst
  serviceAccountName: default
  serviceAccount: default
  nodeName: ip-192-168-7-64.us-west-2.compute.internal
  securityContext: {}
  schedulerName: default-scheduler
  tolerations:
    - key: node.kubernetes.io/not-ready
      operator: Exists
      effect: NoExecute
      tolerationSeconds: 300
    - key: node.kubernetes.io/unreachable
      operator: Exists
      effect: NoExecute
      tolerationSeconds: 300
  priority: 0
  enableServiceLinks: true
```

**Comment by erismaster** [comment]
Hello! I am from the Bug Triage team! This issue has not been updated for a long time, so I'd like to check on the status. The code freeze is starting March 9th, 2021 (about 1 week from now). We want to ensure that each PR has a chance to be merged before code freeze.
As the Issue is tagged for 1.21, is it still planned for this release?

**Comment by erismaster** [comment]
@jayunit100 It looks like #97809 resolves this issue, if thats the case can you close this issue? Thanks!

**Comment by erismaster** [comment]
Moving to 1.22.

/milestone v1.22

**Comment by voigt** [comment]
Hey there (@jayunit100 ), Bug-Triage here 👋 ,
looks like this issue hasn't received any update since it has been assigned to `milestone 1.22`. Given the fact that the code freeze for this release already started, do you think this one is still realistic for 1.22?

From what I get from the comments this ticket likely resolved in #97809. Maybe we can even close the issue? Would be great if anyone could confirm.

In any case, we will clear the milestone label if we do not receive any update within the next 7d.
